.gitignore

```diff
@@ -1,27 +1,13 @@
 /app/config/parameters.yml
-/app/data/images/*
-!/app/data/images
-!/app/data/images/.gitkeep
+/app/data/images/
```

Comment:
Why remove the gitkeep? I like to be able to understand the directory structure of the project in git. This is true for the other directories like this one.